pyshop/shopcore/shop_api.py

```diff
@@ -21,6 +21,8 @@ def get_attribute_value(shop_api, object_name, object_type, attribute_name, data
             value = None
     elif datatype == 'string':
         value = shop_api.GetStringValue(object_type, object_name, attribute_name)
+    elif datatype == 'string_array':
```

Comment:
Bør det også finnes en set-funksjon for "string_array"? Det finnes kanskje bare output atributter nå, men det kan jo legges in for "completeness"

Reply:
Fikset - eksisterer foreløpig ikke i SHOP-kjerne, men det er ok. Ved behov implementeres den der også.